### DISCUSSION TITLE
Page event onLoad

### DISCUSSION THREAD

**Discussion by kemeris2000** [illacloud/illa-builder#1544] [English Support]
Is it possible to run some actions on page load event?

**Comment by AruSeito** [comment]
We have scheduled this feature and it is currently in the final testing phase. It will be launched soon, please see [ILLA Roadmap](https://github.com/orgs/illacloud/projects/4?pane=issue&itemId=25448898) for details

**Comment by kemeris2000** [comment]
Thank you for reply.

**Reply by AruSeito** [reply]
@kemeris2000 Hello, we now support this.
